modules/global-classes/module.php

```diff
@@ -63,4 +65,12 @@ private function add_packages( $packages ) {
 
 		return array_merge( $packages, self::PACKAGES );
 	}
+
+	private function add_env_var( $config ) {
```

Comment:
```suggestion
	private function add_package_env_vars( $config ) {
```

maybe something like this?

also the other one